app/script/localization/Localizer.js

```diff
@@ -70,6 +78,53 @@ class Localizer {
 
 z.localization.Localizer = new Localizer();
 
+z.l10n = (() => {
+
+  function replaceWithString(string, substitute) {
+    return string.replace(/{{\w+}}/, substitute);
+  }
+
+  function replaceWithObject(string, substitute) {
+    for (let identifier in substitute) {
+      if (substitute.hasOwnProperty(identifier)) {
+        string = string.replace(`{{${identifier}}}`, substitute[identifier]);
+      }
+    }
+    return string;
+  }
+
+  return {
+
+    /**
+     * Retrieve localized string and replace placeholders
+     *
+     * This method give you two options to replace placeholders
+     *
+     * @example using a string as substitute
+     * z.l10.text('Hey {{name}}', 'Tod') // returns 'Hey Tod'
+     *
+     * @example using an object as substitute
+     * z.l10.text('{{greeting}} {{name}}', {name: 'Tod', greeting: 'Hey') // returns 'Hey Tod'
+     *
+     * @param {Observable|string} value
+     * @param {string|Object} substitute
+     * @returns {string}
+     */
+    text(value, substitute) {
```

Comment:
@lipis Not talking about the substitute but the source string. Currently we pass in the string identifier to retrieve the localized string. This just takes a string so we have to get the localized variant of the string in another step.